test/libsolidity/syntaxTests/literalSuffixes/suffixableLiterals/invalid_suffixed_integer_too_large.sol

```diff
@@ -5,7 +5,7 @@ contract C {
     uint y = 999999999999999999999999999999999999999999999999999999999999999999999999999999 suffix;
 }
 // ----
-// TypeError 5503: (92-106): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function.
-// TypeError 8838: (92-106): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 5503: (121-206): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function.
-// TypeError 8838: (121-206): The type of the literal cannot be converted to the parameter of the suffix function.
+// TypeError 5503: (92-106): This number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function. The mantissa is out of range of the largest supported integer type.
+// TypeError 8838: (92-106): The number is out of range of type uint256 accepted by the suffix function.
```

Comment:
The first message here should not appear when the suffix function has only one argument.